docs/01-readme.md

```diff
@@ -29,7 +29,9 @@ pass arguments to `cmake`, please use the `CMAKE_ARGS` environment variable. For
 instance:
 
     CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/opt/awesome" make
-
+    
+cmake argument for debian:
+`"-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DSYSCONFDIR=/etc"`
```

Comment:
I'm not sure about this. I would prefer if this were trimmed down to the necessary options and these options are also explained. However, do we want/need that? Isn't it enough to have them in the list below (where they IMO do not need to be explained)?

Reply:
i really don't know the necessary options, i just pasted what i saw in the example on #3106

Reply:
@psychon I don't think this addition belongs here since there already a section on how to install awesome on debian.

Maybe adding this `make` command as part of the `Debian-based` section as part of the `Installing current git master as a package receipts` section ? Something like this
```
sudo apt build-dep awesome
git clone https://github.com/awesomewm/awesome
cd awesome
make package CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DSYSCONFDIR=/etc"
sudo apt install *.deb
```
(similar to what is already done in this commit)

I still think it's better to remove redundant cmake parameters. Maybe do trial and error on a debian machine to see when it works ?